cmd/snap-confine/spread-tests/regression/lp-1599608/task.yaml

```diff
@@ -17,7 +17,7 @@ prepare: |
     cd /
     echo "Install hello-world"
     snap install hello-world
-    systemctl stop snapd.service snapd.socket
+    systemctl stop snapd.refresh.timer snapd.service snapd.socket
```

Comment:
Indeed, I think we do not really need it, I will remove (but waiting for tests first).